operator/charts/netchecks/README.md

```diff
@@ -0,0 +1,61 @@
+# netchecks
+
+![Version: 0.1.2](https://img.shields.io/badge/Version-0.1.2-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 0.3.0](https://img.shields.io/badge/AppVersion-0.3.0-informational?style=flat-square)
+
+Netchecks proactively verifies whether your security controls are working as intended.
+
+## Maintainers
+
+| Name | Email | Url |
+| ---- | ------ | --- |
+| Brian Thorne | <brian@hardbyte.nz> |  |
+
+## Source Code
+
+* <https://github.com/netchecks/operator>
+
+## Requirements
+
+| Repository | Name | Version |
+|------------|------|---------|
+| https://kyverno.github.io/policy-reporter | policy-reporter | 2.14.1 |
+
+## Values
+
+| Key | Type | Default | Description |
+|-----|------|---------|-------------|
+| fullnameOverride | string | `""` |  |
+| installCRDs | bool | `true` |  |
+| nameOverride | string | `""` |  |
+| networkPolicy.enabled | bool | `true` |  |
+| operator.affinity | object | `{}` |  |
+| operator.image.pullPolicy | string | `"IfNotPresent"` |  |
+| operator.image.repository | string | `"ghcr.io/hardbyte/netchecks-operator"` |  |
+| operator.image.tag | string | `""` |  |
+| operator.imagePullSecrets | list | `[]` |  |
+| operator.nodeSelector | object | `{}` |  |
+| operator.podAnnotations."prometheus.io/path" | string | `"/metrics"` |  |
+| operator.podAnnotations."prometheus.io/port" | string | `"9090"` |  |
+| operator.podAnnotations."prometheus.io/scrape" | string | `"true"` |  |
+| operator.podSecurityContext | object | `{}` |  |
+| operator.resources | object | `{}` |  |
+| operator.securityContext | object | `{}` |  |
+| operator.service.port | int | `8080` |  |
+| operator.service.type | string | `"ClusterIP"` |  |
+| operator.serviceAccount.annotations | object | `{}` |  |
+| operator.serviceAccount.create | bool | `true` |  |
+| operator.serviceAccount.name | string | `"netcheck-operator"` |  |
+| operator.tolerations | list | `[]` |  |
+| policyreporter.enabled | bool | `false` |  |
+| probeConfig.affinity | object | `{}` |  |
+| probeConfig.image.pullPolicy | string | `"Always"` |  |
+| probeConfig.image.repository | string | `"ghcr.io/hardbyte/netchecks"` |  |
+| probeConfig.image.tag | string | `""` |  |
+| probeConfig.imagePullSecrets | list | `[]` |  |
+| probeConfig.nodeSelector | object | `{}` |  |
+| probeConfig.podAnnotations | object | `{}` |  |
+| probeConfig.resources | object | `{}` |  |
+| probeConfig.tolerations | list | `[]` |  |
```

Comment:
I assumed with artifacthub etc projects are moving away from duplicating these long tables of configurable values?

I'd rather document in the chart's values.yaml file.

Reply:
Yea artifacthub does this already, so it's not that useful if someone is already viewing it there